### ISSUE TITLE
Only give the interval tool the "large" interactive highlight

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/center-and-variability#399]
**Test device**
MacBook Air 

**Operating System**
13.4

**Browser**
safari

**Problem description**
For https://github.com/phetsims/qa/issues/964, on the Variability screen, the interval tool has two different interactive highlights, depending on whether the mouse is in the play area or in the panel. To me, this is confusing. Also, the smaller highlight isn't part of the focus order as I tab through the screen. Spoke with @catherinecarter and she feels there should probably be only one larger highlight. 

<details>
  <summary>slack conversation</summary>

From Slack:
Nancy Salpepi
  [4:49 PM](https://phetsims.slack.com/archives/D041TMHAXUN/p1690231789141179)
One for question…with interactive highlights on, if I am “on the field” and move the entire interval tool all at once, it highlights both the bottom and the panel. If I do the same in the panel, it only highlights in the panel. Intentional?

Catherine Carter
  [4:50 PM](https://phetsims.slack.com/archives/D041TMHAXUN/p1690231842585479)
I don’t think so, but let me think through…
[4:52](https://phetsims.slack.com/archives/D041TMHAXUN/p1690231920836489)
If the focus is on the play area, the interval tool seems like it makes sense to have the entire rectangle highlight since the abstract representation is modeling the actual data in the play area. Connecting these makes sense. If I’m in the abstract representation only, would it make sense to only focus on the abstract representation? What do you think?

Nancy Salpepi
  [4:54 PM](https://phetsims.slack.com/archives/D041TMHAXUN/p1690232093634349)
Moving the interval tool in the abstract area also moves it in the play area. To me, it seems kinda weird to have 2 different highlights. Also, if I tab around, only the first large highlight is included in the focus order.

Catherine Carter
  [4:56 PM](https://phetsims.slack.com/archives/D041TMHAXUN/p1690232218768059)
I think you’re right, especially since the tab goes to the entire thing. It should probably highlight the entire tool every time. If it were different, it seems like it would inspire a student to wonder why the highlight is different. Since there’s no pedagogical reason for the rectangle to have a different highlight, it should probably be the whole thing.

</details> 

**Steps to reproduce**
1. Turn on interactive highlights
2. on the Variability Screen, check the Interval tool 
3. Move your mouse over the interval in the play area and then in the panel.

**Visuals**
<img width="1440" alt="Screenshot 2023-07-24 at 4 47 55 PM" src="https://github.com/phetsims/center-and-variability/assets/87318828/9be6e23e-2ee9-4228-9d28-6e45fed9a07b">

<img width="1440" alt="Screenshot 2023-07-24 at 4 47 47 PM" src="https://github.com/phetsims/center-and-variability/assets/87318828/fa957962-e39e-4e78-b7b0-4f0193b5de47">

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Center and Variability‬
URL: https://phet-dev.colorado.edu/html/center-and-variability/1.1.0-dev.12/phet/center-and-variability_all_phet.html
Version: 1.1.0-dev.12 2023-07-24 17:55:50 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.5 Safari/605.1.15
Language: en-US
Window: 732x394
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0 (1.0)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by marlitas** [comment]
I saw you switched between the suggestion and bug labels, but I think this is a bug. We definitely didn't actively make this choice. Labeling it as such and removing the second interactive highlight.

**Comment by marlitas** [comment]
Accidentally committed to #407. I disabled the interactive highlight for the Interval Tool in the PlotNode, so now we don't get doubles. However, I feel like we'll want the full highlight to still show up when we hover over the tool in the plot node. This is where I got stuck since the shape changes and I could not find a way to access InteractiveHighlight after construction. Marking for help wanted.
